### PR TITLE
theme: filterable dynamic options.

### PR DESCRIPTION
**Pull request by suraj-webkul**

### CODE REVIEW

**Comment by ashishkumar-webkul** [comment]
@suraj-webkul 

There are some conflicts in this PR. Please fix the PR conflicts.

**Comment by suraj-webkul** [comment]
Resolved conflicts.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Admin/src/Resources/views/settings/themes/edit/product-carousel.blade.php [hunk 1]
```diff
@@ -415,6 +449,56 @@ class="px-3 py-1.5 bg-blue-600 border border-blue-700 rounded-md text-gray-50 fo
...
+                                        'id': 0,
+                                        'name': '@lang('admin::app.settings.themes.edit.no')',
+                                    },
+                                    {
+                                        'id': 1,
+                                        'name': '@lang('admin::app.settings.themes.edit.yes')',
+                                    },
+                                ],
+                            }];
+
```
[the remark this answers is not in the document]
Comment (suraj-webkul) [reply packages/Webkul/Admin/src/Resources/views/settings/themes/edit/product-carousel.blade.php]: done

**Review by devansh-webkul — DISMISSED** [review]

**Comment by ashishkumar-webkul** [comment]
Checked and Found that the filters are working fine, please check the video for reference. 

**Video**

https://github.com/bagisto/bagisto/assets/92721837/60378deb-6d28-4b6f-a9a5-1bc4e9268ac2

**Comment by ashishkumar-webkul** [comment]
Need to fix the issue of static content view, page is not appearing properly. Please fix the issue.

**Video**

**Comment by devansh-webkul** [comment]
Conflicts.

**Comment by ashishkumar-webkul** [comment]
Checked and found that the issue related to the static content pages in theme section and also the product crousal concern are added with filters, hence the issue has been fixed and working fine. Please check the video for reference.

**Video**


https://github.com/bagisto/bagisto/assets/92721837/ce1b4530-e07a-4726-9177-ed4163f4a089

**Review by ashishkumar-webkul — DISMISSED** [review]
checked and found that he static content issue fixed and also product crousal is added with the attribute options.

**Review by devansh-webkul — APPROVED** [review]

**Review by devansh-webkul — APPROVED** [review]

**Pull request merged**